src/plugins/discover/public/application/main/hooks/use_inspector.ts

```diff
@@ -31,11 +37,24 @@ export function useInspector({
   const onOpenInspector = useCallback(() => {
     // prevent overlapping
     setExpandedDoc(undefined);
-    const session = inspector.open(inspectorAdapters, {
-      title: savedSearch.title,
-    });
+
+    const requestAdapters = inspectorAdapters.lensRequests
```

Comment:
This doesn't feel like a great solution -- creating `AggregateRequestAdapter` to support inspecting requests from multiple `RequestAdapter`s. I feel like ideally we would allow customizing the adapters passed to the Lens embeddable, or modify `src/plugins/inspector/public/views/requests/components/requests_view.tsx` to accept multiple `RequestAdapter`s, but both those solutions felt out of scope of this PR.